tests/e2e/project/issues.spec.ts

```diff
@@ -55,3 +55,53 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
   await page.getByRole("button", { name: "Comment" }).click();
   await expect(page.getByText("Issue editing failed")).toBeVisible();
 });
+
+test("go through the entire ui issue flow", async ({
+  page,
+  authenticatedPeer,
+}) => {
+  await authenticatedPeer.startHttpd(8070);
+  await authenticatedPeer.startNode();
+
+  const { rid } = await createProject(authenticatedPeer, "commenting");
+
+  await page.goto(`/seeds/127.0.0.1:8070/${rid}`);
+  await page.getByRole("link", { name: "0 issues" }).click();
+  await page.getByRole("link", { name: "New issue" }).click();
+  await page.getByPlaceholder("Title").fill("This is a title");
+  await page
+    .getByPlaceholder("Write a description")
+    .fill("This is a description");
+
+  await page.getByPlaceholder("Add assignee").fill(authenticatedPeer.nodeId);
+  await page.getByPlaceholder("Add assignee").press("Enter");
+
+  await page.getByPlaceholder("Add tag").fill("bug");
+  await page.getByPlaceholder("Add tag").press("Enter");
+  await page.getByPlaceholder("Add tag").fill("documentation");
+  await page.getByPlaceholder("Add tag").press("Enter");
+
+  await page.getByRole("button", { name: "Submit" }).click();
+
+  await page.getByRole("button", { name: "edit" }).first().click();
+  await page.getByPlaceholder("Title").fill("This is a new title");
+  await page.getByRole("button", { name: "save" }).click();
+
+  await page.getByPlaceholder("Leave your comment").fill("This is a comment");
+  await page.getByRole("button", { name: "Comment" }).click();
+
+  await page.getByRole("button", { name: "reply" }).click();
+  await page.getByPlaceholder("Leave your reply").fill("This is a reply");
+  await page.getByRole("button", { name: "Reply", exact: true }).click();
```

Comment:
We should add some `expect`s for the title, description, assignee, tag, new title, comment and comment reply.